examples/rsp-next-ts/pages/index.tsx

```diff
@@ -162,6 +163,21 @@ export default function Home() {
                 <Item key="copy">Copy</Item>
                 <Item key="paste">Paste</Item>
                 <Item key="replace">Replace</Item>
+                <SubmenuTrigger>
```

Comment:
<img width="370" alt="Screenshot 2024-02-05 at 4 41 48 PM" src="https://github.com/adobe/react-spectrum/assets/44729383/3a49be95-79a2-4aac-a2cf-977a6716cbd6">

for some reason i can't see the chevron's in the next.js app? seems fine in the cra one tho

Reply:
Weird, and they look fine in the collapsible table.

Reply:
Maybe we can merge the example apps and investigate in a follow-up

Reply:
Looks like it was just due to the Next.js app having `box-sizing: border-box;` in its styles